.travis.yml

```diff
@@ -40,8 +42,6 @@ script:
     - lcov --version | grep "1.10"
     - export CC="gcc-5"
     - ./configure --enable-coverage && make && make check && make distcheck && make check-valgrind
-    - clang-format-3.8 -i $(find . -name '*.[ch]') && git diff --exit-code
```

Comment:
if you have to remove this, you need to do so for a good reason, and state exactly in what case it is broken and why that shouldn't be fixed in clang-format instead upstream.

Disabling this without further comment just means you've got style issues and you don't want to fix them, which is OK for WIP patches but they should get fixed nonetheless :)

Reply:
Actually I did this on purpose to have this discussion. clang-format does ugly things like [this](https://github.com/clearlinux/clr-boot-manager/pull/75/files#diff-bd3215aa838aba5c7f9c9b68fe10caddL539).

I do want to question use of automated formatting tool with 8-space indent, ~20 char long identifiers, and line width of 80. And no, I don't think I have issues with style. I do take an issue with what clang-format [wants me to do](https://travis-ci.org/clearlinux/clr-boot-manager/builds/271098489#L2057).

Reply:
clang-format will do ugly things at times. I think it is reasonable to consider moving the 80 character line width and 8 space indent. At this point it is just a simple matter of tweaking some configuration files and rerunning clang-format once we have a consensus so the bike shed is the painful part.

How about 4 space indent and 100 character line width? (I am not really concerned on the indent but don't want line width to stretch too far, for my personal preferences)

Reply:
Well, yes, it's a bikeshed discussion. I simply re-enabled it and made it happy for the sake of this PR. The code in general have a formatting/style issue IMHO, but it should be addressed later by an inspired volunteer (like me on a weekend).

Having said that I'm fine with either 80 or 100 char width. 100 is OK even on a vga console. I'd suggest that we use 4-space indent. Kernel-style indenting does not look good when long identifiers are used for variables and functions.

If we agree on this, I'll file an issue and fix it in its own PR.